zerver/actions/message_delete.py

```diff
@@ -46,56 +44,6 @@ def check_update_first_message_id(
     send_event_on_commit(realm, stream_event, users_to_notify)
 
 
-def _process_grouped_messages_deletion(
```

Comment:
Let's try to do these moves in a prep commit with no functional changes; I think the first two prep commits plus that move could be merged early with a prep PR.